src/Vegetation/PlantHydraulics.jl

```diff
@@ -113,18 +113,22 @@ $(DocStringExtensions.FIELDS)
 struct PlantHydraulicsParameters{FT <: AbstractFloat, PSE}
     "RAI, SAI, LAI in [m2 m-2]"
     area_index::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
-    "water conductivity in the different plant compartments (m/s) when pressure is zero, a maximum"
-    K_sat::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
-    "van Genuchten parameter"
-    vg_α::FT
-    "van Genuchten parameter"
-    vg_n::FT
-    "van Genuchten parameter"
-    vg_m::FT
-    "porosity"
-    ν::FT
-    "storativity"
-    S_s::FT
+    "water conductivity in the above-ground plant compartments (m/s) when pressure is zero, a maximum"
+    above_ground_plant_K_sat::NamedTuple{(:stem, :leaf), Tuple{FT, FT}}
+    "water conductivity in the roots (m/s) when pressure is zero, a maximum"
+    K_sat_root::Function
+    "van Genuchten parameter (1/m)"
+    plant_vg_α::FT
```

Comment:
since these are will within the planthydraulics parameter set, may be ok to just call vg_alpha, vg_n, etc?

Reply:
You mean we can write vg_alpha, vg_n as they are inside the PlantHydraulics.jl file, but in the coupled tests driver files, we keep plant_vg_α and soil_vg_α to differentiate them?